jablib/build.gradle.kts

```diff
@@ -106,6 +106,8 @@ dependencies {
 
     implementation("com.fasterxml:aalto-xml")
 
+    implementation("org.hisp.dhis:json-tree")
```

Comment:
Why another json library?

Reply:
> Why another json library?

Because I know the author personally. And it is fast

Reply:
> For optionals use kongs unirest json

Show me. 😅